unity2ros:
  ros__parameters:
    ROS_IP: "0.0.0.0"

planner:
  ros__parameters:
    rate: 3
    autodepth: 
      kp: 0.4
      ki: 0.8
      kd: 0.1
      min_depth: -0.3
      max_depth: -1.0

recorder:
  ros__parameters:
    width: 1280
    height: 720
    frame_rate: 30
    video_dir: "/home/ros/recordings"

localiser:
  ros__parameters:
    rate: 10
    map_dir: "/home/ros/maps"

tracker:
  ros__parameters:
    rate: 5
    classes_path: "build/nemo/build/lib/nemo/data/classes.txt"
    model_path: "build/nemo/build/lib/nemo/data/model.onnx"
    image_size:
      x: 640
      y: 640